### DISCUSSION TITLE
2024.03.18 회의

### DISCUSSION THREAD

**Discussion by yusoonsun** [bitcamp-teams/mos#20] [회의록]
**참석자 : 김준회, 최성원, 박준수, 권채린, 김선종, 유순선**

# 주제: 스터디 커뮤니티 사이트(모두의 스터디; 모스)

---

## 🧠 UI 프로토 타입

- 스터디 희망지역(3~4군데) & 닉네임 회원 가입할 때 기입..? 회원가입 페이지 구현 필요
- 마이스터디 수정
- 웹 사이트 사이즈 (1440*960)

[Uploading 2024.03.18_회의.pdf…]()

**Comment by joonhoekim** [comment]
![모두의스터디_screenflow_ver1](https://github.com/bitcamp-teams/mos/assets/34194400/63119197-7499-4c50-a863-5a56be91dc8b)
